### ISSUE TITLE
Baby steps with Node JS

### ISSUE THREAD

**Issue by fernandofreamunde** [fernandofreamunde/challenge-github-blog#4]
Node.js is a popular open-source, cross-platform runtime environment for executing JavaScript code outside of a web browser. If you're new to Node.js, here's a simple guide to help you get started:

Install Node.js: The first step to using Node.js is to install it on your computer. You can download the installation file from the official Node.js website. Follow the instructions provided to complete the installation process.

Learn the basics: Once you have installed Node.js, it's time to learn the basics. Start by learning about JavaScript, as Node.js is essentially JavaScript that can run on a server. You should also learn about Node.js modules and how to use them.

Create a Hello World program: A Hello World program is a simple program that outputs the text "Hello, World!" to the console. Create a new file and add the following code:

```js
console.log("Hello, World!");
```

Save the file as "helloworld.js".

Run the program: To run the program, open a terminal or command prompt, navigate to the directory where you saved the file, and type the following command:

```bash
node helloworld.js
```

This will execute the program and output the text "Hello, World!" to the console.

Create a web server: Node.js can also be used to create web servers. Create a new file and add the following code:

```js
const http = require('http');

const hostname = '127.0.0.1';
const port = 3000;

const server = http.createServer((req, res) => {
  res.statusCode = 200;
  res.setHeader('Content-Type', 'text/plain');
  res.end('Hello, World!\n');
});

server.listen(port, hostname, () => {
  console.log(`Server running at http://${hostname}:${port}/`);
});
```

Save the file as "server.js".

Run the web server: To run the web server, open a terminal or command prompt, navigate to the directory where you saved the file, and type the following command:

```bash
node server.js
```

This will start the server and output the message "Server running at http://127.0.0.1:3000/" to the console. Open a web browser and navigate to "http://127.0.0.1:3000/" to see the "Hello, World!" message displayed in the browser.

In conclusion, Node.js is a powerful runtime environment that allows you to run JavaScript code outside of a web browser. By following the steps outlined above, you can get started with Node.js and start building your own JavaScript applications and web servers.
